liferay-echo/liferay-portal
# LPS-197327 Add version tab to web content info panel

Motivation
=======
Add version tab to web content info panel
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-197327)


Steps to Verify:
----------------
1. Set feature.flag.LPS-197307=true
2. Create one web content
1. Edit the web content and save it to create as many versions as we like
1. Go to the web content lists, select a web content and open its info panel
1. Check that the version tab appears and shows the web content versions. We have added the status label for each version even though it was not in the requirements but we talked to Carol and we all agree on it.
1. Create another web content
2. In the list of the web contents select both and check that the info panel shows the detail tab with the new styles
3. Unselect the web web contents without closing the info panel and check that the tab has the new styles
4. Check that everything works without the feature flag


Screenshots (optional):
-----------------------

![Captura de pantalla 2023-10-02 a las 9 37 39](https://github.com/liferay-echo/liferay-portal/assets/3276218/b21c91ce-b61d-4740-90d6-d053fbb869c4)
